File: app/src/main/java/com/saizad/mvvmexample/components/auth/splash/SplashViewModel.java
package com.saizad.mvvmexample.components.auth.splash;


import com.saizad.mvvmexample.di.auth.AuthEnvironment;
import com.saizad.mvvmexample.components.auth.AuthViewModel;

import javax.inject.Inject;

public class SplashViewModel extends AuthViewModel {

    @Inject
    public SplashViewModel(AuthEnvironment authEnvironment) {
        super(authEnvironment);
    }
}

File: app/src/main/java/com/saizad/mvvmexample/components/auth/splash/SplashViewModel.kt
package com.saizad.mvvmexample.components.auth.splash

import com.saizad.mvvmexample.components.auth.AuthViewModel
import com.saizad.mvvmexample.di.auth.AuthEnvironment
import dagger.hilt.android.lifecycle.HiltViewModel
import javax.inject.Inject

@HiltViewModel
class SplashViewModel @Inject constructor(
    authEnvironment: AuthEnvironment
) : AuthViewModel(authEnvironment)

